src/index.ts

```diff
@@ -83,17 +83,21 @@ type PlaywrightTestFixtures = {
 
 const fixtures = baseFolio.extend<PlaywrightTestFixtures, PlaywrightWorkerFixtures, PlaywrightParameters>();
 fixtures.browserName.initParameter('Browser type name', (process.env.BROWSER || 'chromium') as 'chromium' | 'firefox' | 'webkit');
-fixtures.headful.initParameter('Whether to run tests headless or headful', process.env.HEADFUL ? true : undefined);
+fixtures.headful.initParameter('Whether to run tests headless or headful', process.env.HEADFUL ? true : false);
 fixtures.platform.initParameter('Operating system', process.platform as ('win32' | 'linux' | 'darwin'));
 fixtures.screenshotOnFailure.initParameter('Generate screenshot on failure', false);
 fixtures.slowMo.initParameter('Slows down Playwright operations by the specified amount of milliseconds', 0);
 fixtures.video.initParameter('Record videos while running tests', false);
 
 fixtures.browserOptions.init(async ({ headful, slowMo }, run) => {
+  let headless;
+  if (!process.env.PWDEBUG && typeof headful !== 'undefined')
```

Comment:
Why do we need this? When I try locally with just `headless:  !headful`, everything works as expected with PWDEBUG=1.

Reply:
My bad, I should have added some context so its easier understandable. In #212 a change was merged that if no headful parameter is specified, `undefined` gets passed for the `headless` option. This fixed the case, that `PWDEBUG` was not working when `headless: false` got passed to Playwright - which sounded like an upstream bug.

I tried to revert the previous change since PW should handle that and it seems to still work fine with the test cases below which all result in a headful instance are passing:

- `PWDEBUG=1 npx folio -j 1 --retries 3 --reporter list try-playwright.spec.ts`
- `HEADFUL=1 npx folio -j 1 --retries 3 --reporter list try-playwright.spec.ts`
- `npx folio -j 1 --retries 3 --reporter list try-playwright.spec.ts --param headful`

So I changed my PR to just revert the previous change.

Reply:
Great, thank you for more details!

Reply:
_Arguably_ this is an upstream bug. From playwright's perspective, why environment variables override explicit configuration?

Reply:
You mean PWDEBUG? Maybe there was a case which I miss but in my mind its expected when the user uses PWDEBUG that its headfull always.

Reply:
Fair. That is actually quite reasonable.